tests/test_parse.py

```diff
@@ -177,7 +177,7 @@ def test_parse_imports_global_module_reference(self):
                                         'IMPORTS '
                                         'a FROM B '
                                         'c, d FROM E global-module-reference '
-                                        'f, g FROM H {iso(1)}; '
+                                        'f, g FROM H {iso(1)} ;'
```

Comment:
Is this change intended?

Reply:
Hi,
I don't think so. I guess it's just a typo. I'll correct it.

Reply:
Done